jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -170,7 +170,7 @@ def _get_real_xy(self, image, x, y, reverse=False):
             # we aren't actually guaranteed to get a SkyCoord out, just for images
             # with valid celestial WCS
             try:
-                link_type = self.get_link_type(image.label)
+                link_type = self.get_link_type(image.label).lower()
```

Comment:
Just a heads up that this snuck in, in case you didn't mean to. But I don't mind.

Reply:
I just put this everywhere that there could be ambiguity, whether or not it currently fixes anything.